handel/src/cli/index.ts

```diff
@@ -47,8 +48,44 @@ function logCaughtError(msg: string, err: Error) {
     }
 }
 
-function logFinalResult(lifecycleName: string, envResults: EnvironmentResult[]): void {
+async function makeSureDeploymentsLogTableExists() {
```

Comment:
Let's move this function into the dynamodb-calls module, I try to keep the CLI module pretty limited to just starting things and handling output and errors.